src/tools/tidy/src/deps.rs

```diff
@@ -90,15 +90,18 @@ const WHITELIST: &[Crate<'_>] = &[
     Crate("crossbeam-epoch"),
     Crate("crossbeam-utils"),
     Crate("datafrog"),
+    Crate("dlmalloc"),
```

Comment:
are these just things "missing" from previous PRs?

Reply:
yes, these just got moved around in the dependency graph so they're needed in more places now